<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogdim</class>
 <widget class="QDialog" name="Dialogdim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/images/images/Files-New-File-icon.png</normaloff>:/images/images/Files-New-File-icon.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>156</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSlider" name="horizontalSliderSX">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="horizontalSliderSY">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumber_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="horizontalSliderSZ">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber_3"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialogdim</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialogdim</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
